packages/blocks/blocksAntd/src/blocks/Selector/Selector.json

```diff
@@ -229,7 +229,11 @@
       "properties": {
         "onChange": {
           "type": "array",
-          "description": "Trigger action when selection is changed."
+          "description": "Trigger actions when selection is changed."
+        },
+        "onSearch": {
+          "type": "array",
+          "description": "Trigger actions when input is changed. The event object is populated with inputVal which is the input value entered."
```

Comment:
"Trigger actions when input is changed. The input value is passed to the _event operator to be used in actions such as search queries."